src/java/org/apache/cassandra/cql3/statements/schema/AlterSchemaStatement.java

```diff
@@ -102,6 +103,9 @@ Set<String> clientWarnings(KeyspacesDiff diff)
 
     public ResultMessage execute(QueryState state, boolean locally)
     {
+        if (!CassandraRelevantProperties.SCHEMA_MODIFICATIONS.getBoolean())
```

Comment:
shouldn't this be part of Cassandra's configuration ? i.e. we should get it from `DatabaseDescriptor`